app/controllers/va_facilities_controller.rb

```diff
@@ -56,6 +58,21 @@ def show
     @created_practices_categories = get_categories_by_practices(created_practices, [])
   end
 
+  def created_practices_count(id)
+    if helpers.is_user_a_guest?
+      Practice.public_facing.get_by_created_facility(id).size
+    else
+      Practice.published_enabled_approved.get_by_created_facility(id).size
+    end
+  end
+  def adopted_practices_count(id)
+    if helpers.is_user_a_guest?
+      Practice.public_facing.get_by_adopted_facility(id).size
+    else
+      Practice.published_enabled_approved.get_by_adopted_facility(id).size
+    end
+  end
+
```

Comment:
I think we can just create an instance variable in whatever method we want to call these in. So something like:

```
@created_practices_count = helpers.is_user_a_guest? ? Practice.public_facing.get_by_created_facility(id).size : Practice.published_enabled_approved.get_by_created_facility(id).size
```

Reply:
We still have to call the methods for each faciilty.. as the index table is generated..  I can turn the methods in to ternary statements if you prefer.

Reply:
If that's the case, can we move these methods to a `va_facilities_helper.rb` file? That way we won't need the `helper_method` calls at the top of the class and we can call them in the view whenever we need to.

Reply:
Yep - that works.... I think I tried the _helper route.. but I must have been calling the "is_user_a_guest?" method wrong... anyway this works and performance is the same.